smart-note/e2e/login/login.e2e-spec.ts

```diff
@@ -3,45 +3,83 @@ import { browser } from 'protractor';
 
 // sleep for demonstration 
 function sleep() {
-    browser.driver.sleep(2000); 
+    browser.driver.sleep(1500); 
   }
 
 describe('Login page', () => {
   let page: Login;
-
+  var loginURL;
   beforeEach(() => {
     page = new Login();
   });
   
-  // display app title
-  it('should get the title', () => {
-    //page.navigateTo();
-    //sleep();    
+  it('should redirect to the login page if trying to access note while not authenticated', ()=> {
+    loginURL = browser.getCurrentUrl();
+    browser.get('/notes/');
+    expect(browser.getCurrentUrl()).toEqual(loginURL);
+  });
+
+  it('should get the title', () => {   
     expect(page.getTitle()).toEqual('SmartNote');
   });
 
-  // get user name
-  it('should get username', () => {
-    //page.navigateTo();
-    var username = page.getUsername().sendKeys('demo3');
+  it('should check invalid username or password', () => {
+    let username = page.getUsername();
```

Comment:
Nice job, this will ensure that unauthorized user is not allowed to access the notes.